torch/_dynamo/eval_frame.py

```diff
@@ -120,27 +119,9 @@ def __call__(self, fn):
         if isinstance(fn, torch.nn.Module):
             mod = fn
             optimized_forward = self(mod.forward)
+            new_mod = copy.copy(fn)
+            new_mod.forward = optimized_forward
 
-            class TorchDynamoNNModuleWrapper:
-                """
-                A wrapper that redirects the forward call to the optimized
-                forward, while for rest it redirects the calls to the original
-                module.
-                """
-
-                def __getattr__(self, name):
-                    return getattr(mod, name)
-
-                def forward(self, *args, **kwargs):
-                    return optimized_forward(*args, **kwargs)
-
-                def __call__(self, *args, **kwargs):
-                    return self.forward(*args, **kwargs)
-
-            new_mod = TorchDynamoNNModuleWrapper()
-            # Save the function pointer to find the original callable while nesting
-            # of decorators.
-            new_mod._torchdynamo_orig_callable = mod
```

Comment:
You forgot to keep this line of code. This is useful for nested torchdynamo optimize calls.